### PR TITLE
Added 3par and primera examples and readme to point scod csp

### PR DESCRIPTION
**Pull request by sneharai4**
Signed-off-by: Sneha Rai <sneha_srai@yahoo.co.in>

### CODE REVIEW

**Comment by drajen** [comment]
@sneharai4 could you please inspect the new examples structure where all the object referencing a CSP secret should be reflecting what we deploy by default, being "primera3par-secret" your case.

File: examples/kubernetes/hpe-3par-primera/secret.yaml [hunk 1]
```diff
@@ -0,0 +1,13 @@
+apiVersion: v1
+kind: Secret
+metadata:
+  name: hpe-secret
```
Comment (drajen) [inline examples/kubernetes/hpe-3par-primera/secret.yaml:4]: Please change this to `primera3par-secret` and have that reflected in the remaining examples.

Comment (sneharai4) [reply examples/kubernetes/hpe-3par-primera/secret.yaml]: @drajen Done.

**Comment by sneharai4** [comment]
> @sneharai4 could you please inspect the new examples structure where all the object referencing a CSP secret should be reflecting what we deploy by default, being "primera3par-secret" your case.

@drajen Done.

**Pull request merged**